docs/site/content/docs/ref-unmanaged-cluster.md

```diff
@@ -48,6 +48,104 @@ using the default `kind` provider
 tanzu unmanaged-cluster create --control-plane-node-count 2 --worker-node-count 3
 ```
 
+## Installing Profiles
+
+_Warning:_ Profiles is an experimental feature. Use with caution.
+
+When creating a cluster, designate `--profile` to automatically install
+a package from an installed package repository.
+The name of the profile must be the fully qualified name of the package in the package repository,
+or a prefix of the package name in the package repository.
+
+So, for example, to install `fluent-bit` during cluster creation,
+run the following
+
+```sh
+tanzu unmanaged-cluster create my-cluster --profile fluent-bit
```

Comment:
We need to use the package's FQN, right? If so, might be good to update these examples with that.

Reply:
Not necessarily: users can provide the prefix to a package FQN (like we do when installing the CNI).

So running the above works and selects the `fluent-bit.community.tanzu.vmware.com` package